motion/README.md

```diff
@@ -0,0 +1,122 @@
+# MOTION
+
+[MOTION](https://github.com/encryptogroup/MOTION) is a framework for secure multi-party computation (MPC) implemented as a C++ library. 
+It currently implements three MPC protocols for two or more parties: arithmetic and Boolean GMW as well as the BMR protocol, 
+which is the multi-party version of the constant-round Yao's garbled circuit protocol.
+All three protocols provide security against all but one passively corrupted parties, i.e., parties are supposed to follow the protocol but try to learn additional information from what they observe in the protocol.
+MOTION allows to convert between all three protocols and provides both high-level interfaces such as secure integers as well as low-level APIs 
```

Comment:
Maybe an improved wording

```suggestion
MOTION allows to freely switch between all three protocols and provides both high-level interfaces such as secure integers as well as low-level APIs 
```

Also, what do you mean by this exactly? In your mult3 examples (https://github.com/encryptogroup/MOTION/blob/dev/src/examples/tutorial/mult3/common/mult3.cpp#L144), there are different branches for the different protocols, so the framework does not seem to have a simple "plug and play" solution for the different protocols. I have to note that I have not yet come around to reading you paper about MOTION, which I'll do in the next few days.